.github/workflows/ci-presubmit.yaml

```diff
@@ -994,83 +994,83 @@ jobs:
           path: /tmp/artifacts/
 
 
-  tests-scenarios:
+  tests-scenarios-acquisition:
```

Comment:
We should mark this file as generated in https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/.gitattributes so that it doesn't show up in the PR review by default :-)

Reply:
(That can be a separate PR)